molecule/INSTALL.rst

```diff
@@ -0,0 +1,25 @@
+*******
+Install
+*******
+
+Requirements
+============
+
+* LXD
+
+or
+
+* Docker Engine
+* docker-py
```

Comment:
`s/docker-py/docker/` (here and below):
- `docker-py` project renamed `docker-py` Python package to `docker`
- `docker-py` Python package is only required when Ansible 2.2 is used:
  - docker-py # Ansible >= 2.2
  - docker<3.0 # Ansible >= 2.3
  - docker>=3.0 # Ansible >= 2.4

Reply:
will fix, thanks